### DISCUSSION TITLE
updateStrategy for elevation layer

### DISCUSSION THREAD

**Discussion by vladimir-rybalko** [iTowns/itowns#2009] [Q&A]
I have a service for returning elevation tiles.
My service can serve tiles from 0 to 21 scales.
In iTowns, I see different strategies for loading tiles at the layer.
Which strategy is right for me?
I tried to deal with **updateStrategy**. I get a different effects for groups `[3, 7, 9]` and `[5, 10, 15, 18]`.

Are there any thoughts on this?

**Comment by alexLuky** [comment]
@ftoromanoff @jailln @mgermerie @gchoqueux 
any thoughts about this topic? It comes from this [issue](https://github.com/iTowns/itowns/issues/2002) and other elevation stuff.

**Comment by mgermerie** [comment]
Hi ! I believe the choice of `updateStrategy` in iTowns will depend on different factors. There is not an absolute "right" or "wrong" choice. (@gchoqueux correct me if I'm wrong)

Setting up groups for the `updateStrategy` allows one thing among others : in the `[5, 10, 15, 18]` configuration you gave, when displaying a zoom 5 tile, all data between zoom 5 and 9 -included- will be fetched and cached. This grants a more fluid transition when you zoom within these values. However, you could see a little break when zooming from levels 9 to 10. At this moment, data from zoom level 10 and above have not yet been fetched, and the break is due to the time it takes to fetch them.

Another option, should you have a fast internet connection, would be to use `STRATEGY_PROGRESSIVE`. By default, this will load the data from a given zoom level only when you display a tile from this zoom level. Without a fast connection though, this would induce the "break" I mentioned earlier each time you zoom to a new level.

The default option, `STRATEGY_MIN_NETWORK_TRAFIC`, aims to have the lowest amount of network traffic off all implemented strategies, and could be a good choice to have your application running decently on any internet conditions.

**Reply by vladimir-rybalko** [reply]
@mgermerie, thanks for the reply. Unfortunately, the `updateStrategy` `[5, 10, 15, 18]` groups doesn't work for me. I found that working option for me `[5, 10, 15]`.
Another problem is the appearance of breaks in the tiles on the base map example. Can you explain why this behavior occurs?
I putting my a code example that leads to this at [codesandbox](https://13r1b7.csb.app/)
![image](https://user-images.githubusercontent.com/3933179/222498071-f02edaf6-c2ca-43e8-bbf0-b14b13775f45.png)
